api/actions/createWorkItem.js

```diff
@@ -23,10 +27,14 @@ module.exports = {
     eventName: GithubEvents.events.ISSUE_LABELED,
     fn: async function (req) {
         const {
-            issue: { labels, url, title, body, milestone }
+            issue: { labels, url, body, milestone }
+        } = req.body;
+        var {
+            issue: { title }
         } = req.body;
         const { config } = req.git2gus;
         const { hideWorkItemUrl } = config;
+        const { suppressGithubComments } = config;
```

Comment:
This confirm to suppress commenting on Github issues, but rather update the description instead. Prevents constant messaging to external customers when they cannot act on those.

Reply:
this is cool